test/optional-validation.test.js

```diff
@@ -308,6 +308,19 @@ describe('optional-validation', function() {
           d.updateAttributes({'name': NEW_NAME}, expectChangeSuccess(done));
         });
       });
+
+      it('returns error when updating id property when forceId is set', function(done) {
```

Comment:
returns an error when trying to update the id property when forceId is set to true